pkg/driver/clients/grpc_client.go

```diff
@@ -57,35 +65,26 @@ func InvokeGRPC(function *common.Function, runtimeSpec *common.RuntimeSpecificat
 	start := time.Now()
 	record.StartTime = start.UnixMicro()
 
-	dialContext, cancelDialing := context.WithTimeout(context.Background(), time.Duration(cfg.GRPCConnectionTimeoutSeconds)*time.Second)
-	defer cancelDialing()
-
 	var dialOptions []grpc.DialOption
 	dialOptions = append(dialOptions, grpc.WithTransportCredentials(insecure.NewCredentials()))
-	dialOptions = append(dialOptions, grpc.WithBlock())
-	if cfg.EnableZipkinTracing {
```

Comment:
Did you move it somewhere else or remove it completely?

Reply:
WithBlock() has been deprecated, since the gRPC API has changed.
https://pkg.go.dev/google.golang.org/grpc@v1.65.0#WithBlock

For Zipkin, I have updated the otelgrpc handler.
https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.53.0#NewClientHandler